src/App.js

```diff
@@ -1,30 +1,28 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { routerParams } from '@red-hat-insights/insights-frontend-components';
```

Comment:
I think that we want to use routerParams here to store current URL and show correct breadcrumbs right?